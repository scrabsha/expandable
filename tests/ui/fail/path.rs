#![allow(unused)]

#[expandable::expr]
macro_rules! a {
    () => {
        path:::<foo>
    };
}

#[expandable::expr]
macro_rules! b {
    () => {
        path::< <foo> >
    };
}

#[expandable::expr]
macro_rules! c {
    () => {
        path::1
    };
}

fn main() {}
