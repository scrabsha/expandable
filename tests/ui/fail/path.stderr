error: Potentially invalid expansion. Expected an identifier, `Self`.
 --> tests/ui/fail/path.rs:6:15
  |
6 |         path:::<foo>
  |               ^

error: Potentially invalid expansion. Expected an identifier, `-`, `>`, a `{`, a literal.
  --> tests/ui/fail/path.rs:13:17
   |
13 |         path::< <foo> >
   |                 ^

error: Potentially invalid expansion. Expected an identifier, `Self`.
  --> tests/ui/fail/path.rs:20:15
   |
20 |         path::1
   |               ^
